src/data/_contributors.yaml

```diff
@@ -1329,3 +1329,19 @@ maciejzasada:
     - author
   google: "104812516714503133849"
   email: maciej@unit9.com
+
+mcasas:
```

Comment:
`_contributors.yaml` needs to be sorted by family name, would you please adjust.